tests/host_task/cuda_host_task_interop_api.cpp

```diff
@@ -0,0 +1,134 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Provide verification for sycl::interop_handle::get_native functions
+//  this test check interop API with CUDA back-end only.
+//
+*******************************************************************************/
+
+#include "../common/common.h"
+
+#ifdef SYCL_BACKEND_CUDA
+
+#include <cuda.h>
+#include <sycl/ext/oneapi/experimental/backend/cuda.hpp>
+
+#endif  // SYCL_BACKEND_CUDA
+
+#define TEST_NAME cuda_host_task_interop_api
+
+namespace TEST_NAMESPACE {
+using namespace sycl_cts;
+
+class TEST_NAME : public sycl_cts::util::test_base {
+  /** return information about this test
+   */
+  void get_info(test_base::info& out) const override {
+    set_test_info(out, TOSTRING(TEST_NAME), TEST_FILE);
+  }
+
+#ifdef SYCL_BACKEND_CUDA
+  CUresult call_cuda(CUcontext ctx, CUdeviceptr mem, size_t size,
+                     uint32_t pattern) {
+    cuCtxSetCurrent(ctx);
+    CUresult ret = cuMemsetD32(mem, pattern, size);
+    if (ret != CUDA_SUCCESS) FAIL(log, "CUDA invocation returned error code.");
```

Comment:
Can we use CHECK macro here? I think FAIL is deprecated.

Reply:
Done